.env-example

```diff
@@ -0,0 +1,3 @@
+REACT_APP_SUBSTRATE_PROVIDER=wss://wss.rialto.brucke.link
+REACT_APP_ETHEREUM_PROVIDER=http://rpc.rialto.brucke.link:8545
+EXPECTED_ETHEREUM_NETWORK_ID=105
```

Comment:
I thought that all envs need to be prefixed with `REACT_APP`, otherwise they are not leaked to the app, no?

Reply:
I think this is when you use create react app. If you use webpack or in our case parcel, It seems not to be a requirement.
I've been fighting with so kind of cache.. and ended up testing both. I'll actually remove the prefix to make those variables shorter.